app.js

```diff
@@ -1,22 +1,27 @@
 const express = require('express')
 const seed = require('./db/seeds/seed')
-const {getAllTopics, getAllApi, getArticleByArticleId, getAllArticles, getArticleIdComments} = require('./controllers/controllers')
+const {getAllTopics, getAllApi, getArticleByArticleId, getAllArticles, getArticleIdComments, postCommentsToArticle} = require('./controllers/controllers')
 
 const app = express()
 
+app.use(express.json())
 app.get('/api/topics',getAllTopics)
 app.get('/api', getAllApi)
 app.get('/api/articles/:article_id',getArticleByArticleId)
 app.get('/api/articles',getAllArticles)
 app.get('/api/articles/:article_id/comments',getArticleIdComments)
+app.post('/api/articles/:article_id/comments',postCommentsToArticle)
+
 
 app.all('/*',(request, response) =>{
     response.status(404).send({ msg: 'URL does not exist'})
   })
   app.use((err, req, res, next) => {
     if (err.code === '22P02'){
      res.status(400).send({msg : 'URL does not exist, the key you gave is not a number - Bad Request!'})
-    } else if (err.status) {
+    }else if (err.code === '23503'){
```

Comment:
Excellent to see you handling this with PSQL instead of a custom handler, superb work